app/assets/stylesheets/modules/_campaign.styl

```diff
@@ -46,3 +46,9 @@
 
     .campaign-use-dates
       display: block
+
+.create-campaign-button
+  margin-bottom: 50px
+
+.wizard .right
+  width: 100% !important
```

Comment:
This seems very dangerous, as `wizard` is used in a lot of places. Can we find a better way to ensure the correct width in this context without `!important`?